components/eam/src/chemistry/mozart/mo_gas_phase_chemdr.F90

```diff
@@ -450,7 +450,7 @@ subroutine gas_phase_chemdr(lchnk, ncol, imozart, q, &
     !        ... Xform from mmr to vmr
     !-----------------------------------------------------------------------      
     call mmr2vmr( mmr, vmr, mbar, ncol )
-
+ 
```

Comment:
You've got another errant whitespace chance here...seems silly, but these are the kinds of things that can cause merge conflicts if/when we try to update to/from e3sm.

Reply:
Woops, didn't notice. I see you fixed it below, thanks!